code/modules/mod/modules/modules_timeline.dm

```diff
@@ -373,6 +373,7 @@
 		qdel(src)
 	else if(timetokill <= 0)
 		to_chat(captured, span_notice("As the last essence of your being is erased from time, you are taken back to your most enjoyable memory. You feel happy..."))
+		to_chat(world, span_notice("As the last vestiges of their being are erased from time, all memories of [captured.real_name] that you might have had fade from your mind!"))
```

Comment:
A to_chat going to the world? You should never be doing that unless you are in some kind of global context such as inside the MC or doing announcements.